### ISSUE TITLE
[Meds / Renewal] Fix inconsistent messaging around renewal flow initiation

### ISSUE THREAD

**Issue by jasonday** [department-of-veterans-affairs/va.gov-team#125848]
## Context
This ticket emerged from staging review and was cut by Jason Day. It was initially given to the Secure Messaging team, before Raul tapped the Medications team in instead, and added it to the meds board, on December 30, 2025.

## Findings Details
- **VA.gov Experience Standard - issue:** [User is uncertain where a button or link leads.](https://design.va.gov/about/experience-standards/#user-knows-where-a-button-or-link-leads)
- **VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
- **Launch-blocking:** No
- **Design System review:** No
- **Collab Cycle Reviewer:** @erinrwhite (IA), @jasonday (Accessibility)

### Background
It may be unclear to users why "Start a new message" appears on some prescriptions, "Send a renewal request message" appears on others, and "Learn how to renew prescriptions" appears on others, and why those links behave differently. 

`Send a renewal request message` opens a modal, which then takes them one step into SM and prepopulates the message text

`Start a new message` goes straight to the first step in the SM flow, does not prepopulate the message text

`Learn how to renew prescriptions` goes to a resources and support page that doesn't mention the `Send renewal request feature` (yet)

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="417" height="668" alt="Image" src="https://github.com/user-attachments/assets/4a8017f8-04da-4f72-8e61-8856d11f5783" />

</details>

## Description
If possible, create one path to start a message that is consistent across prescription types. In instances where there are links to "Learn more", clarify why that link is there rather than a link to send a message. 

Finally, on the [Resources and Support page for Renewing a prescription](https://staging.va.gov/resources/how-to-renew-a-va-prescription), ensure content includes this new path to send a message.

## Notes
- We're assuming that the flag we're using to determine whether or not "discontinued" text should be shown can also be used to 

## Resources
<img width="706" height="545" alt="Image" src="https://github.com/user-attachments/assets/6307639d-64b4-4624-8cb4-36f920b7501e" />

## References
- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** User flow

## Acceptance Criteria
- [ ] Renewal flow initiation has been made more consistent
- [ ] Code review

**Comment by michaelbrodsky-a6** [comment]
**Note from Impromptu Refinement on Fri, Jan 9, 2026:** The work in this ticket is being folded into ticket [#41284](https://github.com/department-of-veterans-affairs/vets-website/issues/41284) (Refactor send renewal messaging logic) instead. So moving this ticket to "Done" and marking it "Close as not planned".
